main.go

```diff
@@ -114,11 +120,19 @@ func kubeconfigPath(m interface{}) (string, func(), error) {
 	return "", cleanupFunc, nil
 }
 
-func kubectl(m interface{}, kubeconfig string, args ...string) *exec.Cmd {
+func kubeconfigContext(m interface{}) string {
```

Comment:
Can you remove this function and just do it in `kubectl()`?

```
func kubectl(m interface{}, ...) *exec.Cmd {
    // ...
    context :=  m.(*config).kubeconfigContext
    if context != "" {
        // ...
    }
    // ...
}
```

Reply:
sure